src/client/initialize.ts

```diff
@@ -59,6 +60,14 @@ export function initialize<T = {}>(): PluginInstance<T> {
     Object.assign(subscribedInteractions, updatedInteractions);
   });
 
+  on('wb:plugin:action-effect:invoke', (id: string) => {
+    const effect = registeredEffects[id];
+    if (!effect) {
+      throw new Error('No effect found.');
```

Comment:
I'm not sure this should be an error -- having an effect configured in the config options, but not setting up a handler and invoking it should probably just do nothing. Maybe we can do a `console.warn` at the most?